ui/app/components/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -45,8 +50,19 @@ export default class TransactionListItemDetails extends PureComponent {
     onRetry(id)
   }
 
+  handleCopyTxId = () => {
+    const { transactionGroup} = this.props
+    const { primaryTransaction: transaction } = transactionGroup
+    const { hash } = transaction
+
+    this.setState({ justCopied: true }, () => copyToClipboard(hash))
+
+    setTimeout(() => this.setState({ justCopied: false }), 1000)
```

Comment:
Should we move this into the callback for the above `setState` call? We'd want this to run after the copy?

Reply:
yea make sense, i will move it

Reply:
updated